internal/checker/cloudformation/rules/iam/verify_policy_not_allow_credential_exposure.rego

```diff
@@ -0,0 +1,76 @@
+# METADATA
+# title: "Verify credentials for IAM policies are not exposed"
+# description: "This policy concerns AWS IAM users who utilize different credentials like passwords or access keys. Certain actions, such as ecr:GetAuthorizationToken and iam:UpdateAccessKey, have the potential to expose these credentials in their API responses."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-managedpolicy.html
+# custom:
+#   id: CB_CFT_84
+#   severity: LOW
+package lib.cloudformation.CB_CFT_84
+import future.keywords.in
+
+is_valid {
+	supported_resources := ["AWS::IAM::Group", "AWS::IAM::Policy", "AWS::IAM::Role", "AWS::IAM::User", "AWS::IAM::ManagedPolicy"]
+	some resources in input.Resources
+	resources.Type in supported_resources
+}
+
+resource[type] {
+	some resource in input.Resources
+    is_valid
+    count(fail) > 0
+    type := resource.Type
+}
+
+resource[type] {
+	some resource in input.Resources
+    is_valid
+    count(pass) > 0
+    type := resource.Type
+}
+
```

Comment:
declare this outside so you can just use it in both of fail block
    action_value := ["ecr:GetAuthorizationToken", etc...]